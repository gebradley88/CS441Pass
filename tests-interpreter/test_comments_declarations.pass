print -2012.120728;
// print 1;
print 2;
/* print 3; */
print 4;
/* 
	multi
	line
	comment
*/
print 6; // comment
int  l = 6;
print l + 1;

int a, b, c;
a = 10;
b = 8;
c = 9;
print b;

float y = -10.09999999, d = 9.0098;
print d;
print y;
