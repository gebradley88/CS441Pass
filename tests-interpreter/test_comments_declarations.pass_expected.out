-2012.120728
2
4
6
7
8
9.009800
-10.100000
